m4_sessions/1705-inning/group_six.md

```diff
@@ -10,5 +10,13 @@
 
 ### Outline: 
 
-<p>10 minutes for answering questions. 15 minutes whiteboard process. 25 minutes group discussions. 10 minutes for conclusion. </p>
+  * Answer question on their own (10 minutes):
+     * What is knowledge?
+     
+  * Whiteboard (15 minutes)
+    * Symbolic logic exercise
```

Comment:
Do you just want students to listen, should they draw it, or should they ask questions?